presto-product-tests/src/main/resources/sql-tests/datasets/csv_with_header_footer.ddl

```diff
@@ -0,0 +1,11 @@
+-- type: hive
+CREATE %EXTERNAL% TABLE %NAME%
+(
+	c_bigint BIGINT,
+	c_varchar VARCHAR(255))
+ROW FORMAT SERDE 'org.apache.hadoop.hive.serde2.OpenCSVSerde'
+STORED AS TEXTFILE
+LOCATION '%LOCATION%'
+TBLPROPERTIES (
+    'skip.header.line.count' = '1',
+    'skip.footer.line.count' = '1')
```

Comment:
I don't see tests that are using these tables. See:
```
./presto-product-tests/src/main/resources/sql-tests/testcases/csv_table_with_custom_parameters.sql
./presto-product-tests/src/main/resources/sql-tests/testcases/csv_table.sql
```

If you provide convention tests like the ones mentioned above then you can remove all select tests from `TestCsv` and leave there only tests for inserts. Or even you could remove tests for insert from there because that could be easily tested in `TestHiveIntegrationSmokeTest`